### ISSUE TITLE
Unable to Restore Wallet

### ISSUE THREAD

**Issue by ngkong** [input-output-hk/cardano-sl#3894]
Currently we are disabling our exchange cardano wallet service because we are unable to migrate wallet from old version to new version (1.3.2)

How we reproduce:
1. Install new cardano-sl based on the guide
2. Run the ./connect-to-mainnet and wait for 100% synced
3. restore wallet with https://localhost:8090/api/v1/wallets endpoint, operation = restore
4. wait for restore sync complete

Actual result:
balance = 0, json:
```
{
  "data": [
    {
      "id": "Axxxxx",
      "name": "hot-storage",
      "balance": 0,
      "hasSpendingPassword": true,
      "spendingPasswordLastUpdate": "2018-11-23T08:35:55.588418",
      "createdAt": "2018-11-23T08:35:55.588418",
      "assuranceLevel": "strict",
      "syncState": {
        "tag": "synced",
        "data": null
      }
    }
  ],
  "status": "success",
  "meta": {
    "pagination": {
      "totalPages": 1,
      "page": 1,
      "perPage": 10,
      "totalEntries": 1
    }
  }
}
```

also when we try to query for transaction history:
```
curl --cacert state-wallet-mainnet/tls/client/ca.crt --cert state-wallet-mainnet/tls/client/client.pem  "https://127.0.0.1:8090/api/v1/transactions?wallet_id=Axxxxx"   | jq
{
  "status": "error",
  "diagnostic": {
    "msg": "Request error (InternalError)"
  },
  "message": "UnknownError"
}
```

Expected result:
balance > 0
normal transaction history

**Comment by ngkong** [comment]
I've tried deleting wallet and call restore wallet api, after wait for 100% sync, still 0 balance. Also tried building cardano-sl on brand new folder, still same.

**Comment by ngkong** [comment]
for comparison the exactly same wallet id (cwId) on old cardano-sl:

![image](https://user-images.githubusercontent.com/22634237/48950470-cd525980-ef6d-11e8-81a0-51052551153b.png)

**Comment by neville-freeman** [comment]
Hello, `ngkong`. We understand this is connected to a Zendesk support ticket. We recommend following up there, and in the Slack communication channel we've provided.

**Comment by rvl** [comment]
Closing this ticket since we resolved the problem in the [2.0.0 release](https://github.com/input-output-hk/cardano-sl/releases/tag/2.0.0).
